.circleci/config.yml

```diff
@@ -162,9 +162,15 @@ jobs:
           command: echo ". $CIRCLE_WORKING_DIRECTORY/girder_env/bin/activate" >> $BASH_ENV
       - restore_cache:
           key: npm-{{ arch }}-{{ checksum "girder/package-lock.json" }}
+      - run:
+          name: Install top level node modules
```

Comment:
Can you change the name to make it clear that this is installing the `@girder/lint` package?